indico/modules/events/roles/templates/roles.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
     <table class="i-table-widget multi-tbody" id="event-roles">
-        {{ render_roles(roles) }}
+        {{ render_roles(roles, collapsed=true, email_button=true) }}
```

Comment:
no need to change anything here when having the correct default values for those two params